test/globals.js

```diff
@@ -28,7 +28,7 @@ function checkExported( assert, methods, isAssertion ) {
 }
 
 QUnit.test( "QUnit object", function( assert ) {
-	assert.ok( QUnit instanceof QUnit.constructor, "Global QUnit built from it's own constructor" );
+	assert.ok( QUnit instanceof QUnit.constructor, "Global QUnit built from its own constructor" );
```

Comment:
This used to be required because of the old-school logging setup (e.g. `QUnit.begin = fn;`). It can be dropped now that we no longer support that setup style.